.htmlhintrc

```diff
@@ -0,0 +1,61 @@
+{
+  /*
+   * STANDARD
+   * =================
+   */
+  
+  // All tagnames must be in lowercase only
+  "tagname-lowercase": true,
+  
+  // All attribute names must be in lowercase
+  "attr-lowercase": true,
+  
+  // Attribute values must be enclosed by double quotemarks
+  "attr-value-double-quotes": true,
+  
+  // Attributes cannot be duplicated on a single tag
+  "attr-no-duplication": true,
+  
+  // Don't look for doctypes in the html files
+  "doctype-first": false,
+  
+  // Tags must be paired
+  "tag-pair": true,
+  
+  // Empty tags must be closed by self
+  "tag-self-close": true 
+  
+  // All special characters must be escaped
+  "spec-char-escape": true,
```

Comment:
I don't believe it forces that, if you look at the rule, it's actually quite simplistic

https://github.com/yaniswang/HTMLHint/blob/master/src/rules/spec-char-escape.js#L12

In fact, I think it deserves another PR upstream to add more special characters that require to be escaped.